### PR TITLE
creating tag view controller

### PR DESCRIPTION
**Pull request by natashaarmbrust**

### CODE REVIEW

File: Podcast/EpisodeTableViewCell.swift [hunk 1]
```diff
@@ -338,13 +341,17 @@ class EpisodeTableViewCell: UITableViewCell {
     }
     
     func didPressMoreButton() {
-        
+        delegate?.episodeTableViewCellDidPressMoreActionsButton(episodeTableViewCell: self)
     }
     
-    func didPressTagButton(button: UIButton) {
+    
+    func didPressFeedControlButton() {
```
Comment (dennisfedorko) [inline Podcast/EpisodeTableViewCell.swift]: what does this do?

File: Podcast/TagViewController.swift [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+//
+//  Created by Natasha Armbrust on 3/12/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class TagViewController: UIViewController, UITableViewDelegate, UITableViewDataSource,RecommendedSeriesTableViewCellDelegate, RecommendedSeriesTableViewCellDataSource, RecommendedEpisodesOuterTableViewCellDelegate, RecommendedEpisodesOuterTableViewCellDataSource, TagTableViewHeaderDelegate {
+    
+    private var activity: NSMutableArray?
```
Comment (dennisfedorko) [inline Podcast/TagViewController.swift]: don't need private or nsmutablearray

File: Podcast/TagViewController.swift [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    let sectionContentIndentifiers = ["SeriesCell", "EpisodesCell"]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    
+        view.backgroundColor = .podcastWhiteDark
+        
+        setupNavigationBar()
+        
+        for i in 0..<sectionNames.count {
```
Comment (dennisfedorko) [inline Podcast/TagViewController.swift:30]: space between 0 ..< sectionNames.coun t

File: Podcast/TagViewController.swift [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+        
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+        tableView.contentInset = UIEdgeInsetsMake(0, 0, appDelegate.tabBarController.tabBarHeight, 0)
+        view.addSubview(tableView)
+        
+        fetchSeries()
+        fetchEpisodes()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
```
Comment (dennisfedorko) [inline Podcast/TagViewController.swift]: if its blank delete it

File: Podcast/TagViewController.swift (hunk 2 above)
Comment (natashaarmbrust) [reply Podcast/TagViewController.swift]: addressed

File: Podcast/TagViewController.swift (hunk 3 above)
Comment (natashaarmbrust) [reply Podcast/TagViewController.swift:30]: i can't have a space b/c thats not swift syntax

File: Podcast/TagViewController.swift (hunk 4 above)
Comment (natashaarmbrust) [reply Podcast/TagViewController.swift]: addressed

**Pull request merged**